blocks/library/code/index.js

```diff
@@ -32,6 +32,10 @@ registerBlockType( 'core/code', {
 		},
 	},
 
+	example: {
+		content: '<?php echo "Beep Beep Boop";?>',
```

Comment:
Very minor: I'd expect a space before the closing `?>`

https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#opening-and-closing-php-tags

Reply:
Minor: PHP Guidelines recommend single quotes:

>If you’re not evaluating anything in the string, use single quotes.

https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#single-and-double-quotes